generateHtmlTemplates.js

```diff
@@ -0,0 +1,55 @@
+const ejs = require("ejs");
+const fs = require("fs-extra");
+const path = require("path");
+
+async function loadTranslations(language, template) {
+  const filePath = path.join(
+    __dirname,
+    "templates/" + template,
+    "/i18n",
+    `${language}.json`
+  );
+  return await fs.readJson(filePath);
+}
+
+async function generateHtmlTemplates() {
+  const languages = ["de"]; // Elenca le lingue supportate
```

Comment:
qui avremo "de", "fr", "sl"

Reply:
dobbiamo aggiungere anche it, corretto? @SarahDonvito la generazione del template base potrebbe comunque passare da qui, ti torna?

Reply:
non so se avrebbe senso dal momento che vorrebbe dire realizzare a mano l'html ita qui e copiarlo in delivery-push